tidy3d/components/source/base.py

```diff
@@ -74,14 +74,15 @@ def plot(
         y: Optional[float] = None,
         z: Optional[float] = None,
         ax: Ax = None,
+        transpose: bool = False,
```

Comment:
I verified that the changes in this file work by verifying `transpose=True` works in [VizSimulation.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/VizSimulation.ipynb) *(and many other tidy3d-notebook examples)*.